### ISSUE TITLE
Refine page template structure for better accessibility and customisation

### ISSUE THREAD

**Issue by domoscargin** [alphagov/govuk-frontend#6456]
### Decide how to resolve "content outside of landmarks" violations

These were [previously reported by axe](https://github.com/alphagov/govuk-frontend/issues/1604) as a WCAG violation, but have now been downgraded to "best practice. This seems to largely apply to anything that normally goes into the `beforeContent` block, which sits outside of any regions.

Example: https://govuk-frontend-review.herokuapp.com/full-page-examples/upload-your-photo-success

### Explore decoupling the `<header>` tag from the GOV.UK Header component

The Header component is currently also a `<header>` element, which means if users want to include anything else inside the `<header>`, they need to reimplement the Header component.

We'd want to spike this, as well as talking to the community about this, and other changes.

Consider moving the `<header>` tag into the page template and placing the Header component within it

### Explore decoupling the `<footer>` tag from the GOV.UK Footer component

Similar to the header above.

We'd want to spike this, as well as talking to the community about this, and other changes.

### Facilitate the implementation of Service navigation through new blocks and/or variables

It'd be nice if the service navigation could be rendered without having to redefine the whole `header` block. Or if we allow users to set a service name via a variable rather than define a Service navigation

### Update page template guidance if necessary

- We may need a content-shaped person

### Non-obvious things we may need to do

- Ask users about their current frustrations with the existing page template
- Communicate with port maintainers about the impact of these changes
- Check the Prototype Kit, Design System website and review app for breaking changes

**Comment by kellylee-gds** [comment]
@Kam-Nijjar this epic is marked as done and closed - can it be removed from the board?

**Comment by Kam-Nijjar** [comment]
@kellylee-gds, this can be closed once the release is done (ready for release [Update page template guidance](https://github.com/orgs/alphagov/projects/53/views/92?pane=issue&itemId=141155266&issue=alphagov%7Cgovuk-design-system%7C5037
